app/src/main/java/com/example/keniphhomemaintenance/database/DwellingsDao.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.keniphhomemaintenance.database
+
+import androidx.room.*
+import com.example.keniphhomemaintenance.dwellings.Dwelling
+import com.example.keniphhomemaintenance.maintenance_screen.MaintenanceItem
+
+@Dao
+interface DwellingsDao {
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    suspend fun addDwellingToDb(dwelling: Dwelling): Long
```

Comment:
Omit "ToDb" from this + other calls; all Dao operations happen on the db so it's redundant

Reply:
lookup how to use varargs in parameters. We can boil these first two functions into one with varargs